pyproject.toml

```diff
@@ -43,6 +43,7 @@ dependencies = [
     "lxml",
     "psutil",
     "requests",
+    "importlib_resources"
```

Comment:
This package should only be installed for older Python, like this: `importlib_resources; python_version < '3.10'`

Reply:
Thanks. I didn't know that pyproject.toml support conditional dependencies. Great feature!